manager/src/main/java/com/redhat/service/bridge/manager/models/ConnectorEntity.java

```diff
@@ -0,0 +1,155 @@
+package com.redhat.service.bridge.manager.models;
+
+import java.time.ZonedDateTime;
+import java.util.Objects;
+import java.util.UUID;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.FetchType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.NamedQueries;
+import javax.persistence.NamedQuery;
+import javax.persistence.Table;
+import javax.persistence.Version;
+
+import org.hibernate.annotations.Type;
+import org.hibernate.annotations.TypeDef;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.redhat.service.bridge.infra.models.dto.ConnectorStatus;
+
+import io.quarkiverse.hibernate.types.json.JsonBinaryType;
+import io.quarkiverse.hibernate.types.json.JsonTypes;
+
+@NamedQueries({
+        @NamedQuery(name = "CONNECTORENTITY.findByProcessorIdAndName",
+                query = "from ConnectorEntity c where c.name=:name and c.processor.id=:processorId"),
+})
+@Entity
+@Table(name = "CONNECTOR_ENTITY")
```

Comment:
I think the DB table can be just call `CONNECTOR`. WDYT?

Reply:
It was renamed `CONNECTORENTITY` to avoid clash with the SDK. Do you mean to change only the table name and and keep the entity name `ConnectorEntity`?

Reply:
Yes. That was my suggestion, but feel free to tell me it's a bad one :-)